app/domain/errors.rb

```diff
@@ -588,6 +588,11 @@ module AuthnJwt
         msg: "Signing key type '{0-type}' is invalid",
         code: "CONJ00121E"
       )
+
+      InvalidSigningKeySettings = ::Util::TrackableErrorClass.new(
+        msg: "Invalid signing key settings: {0-validation-error}",
+        code: "CONJ00122E"
+      )
```

Comment:
1. If we replaced CONJ00086. Can we remove it?
2. Why we adding new error instead of using the old one ?

Reply:
What do we prefer to change error message and to leave error code as is :
```
      InvalidUriConfiguration = ::Util::TrackableErrorClass.new(
        msg: "Signing key URI configuration is invalid",
        code: "CONJ00086E"
      )
```
or to introduce completely new error.
As you can see the old error text is too specific for new variables mixture.
CC: @szamir1

Reply:
I prefer not having old Errors that not in use.
From my side each way is o.k. removing/updating the old one..

Reply:
I've removed old message